libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -267,7 +265,9 @@ auto start([[maybe_unused]] irods::default_re_ctx& re_ctx, const std::string& _i
 		std::uint64_t time_ms = ts_clock::now().time_since_epoch() / std::chrono::milliseconds(1);
 		json_obj["@timestamp"] = time_ms;
 
-		insert_or_parse_as_bin(json_obj, "hostname", host_name, time_ms);
+		std::string host_name = boost::asio::ip::host_name();
```

Comment:
Should we consider using `boost::asio::ip::host_name()` in other places throughout iRODS, or is this specific to AMQP interactions?

Reply:
`boost::asio::ip::host_name()` is a good C++-style replacement for `gethostname()` in most cases. So yeah I think it should be considered.

Reply:
Agreed